bedrock-devnet/devnet/__init__.py

```diff
@@ -97,9 +107,9 @@ def main():
       sdk_addresses_json_path=pjoin(devnet_dir, 'sdk-addresses.json'),
       rollup_config_path=pjoin(devnet_dir, 'rollup.json'),
       fork_public_network = args.fork_public_network,
```

Comment:
Can we remove this empty space?  ` = ` -> `=`

Reply:
Updated